generatebundlefile/hack/generate_bundle.sh

```diff
@@ -35,6 +35,17 @@ mkdir -p ${BASE_DIRECTORY}/bin
 make oras-install
 
 ECR_PASSWORD=$(aws ecr-public get-login-password --region us-east-1)
-cd output/
+cd ${BASE_DIRECTORY}/generatebundlefile/output
 ${BASE_DIRECTORY}/bin/oras push -u AWS -p "${ECR_PASSWORD}" "${IMAGE_REGISTRY}/eks-anywhere-packages-bundles:v1-21-${CODEBUILD_BUILD_NUMBER}" bundle.yaml
-${BASE_DIRECTORY}/bin/oras push -u AWS -p "${ECR_PASSWORD}" "${IMAGE_REGISTRY}/eks-anywhere-packages-bundles:v1-21-latest" bundle.yaml
\ No newline at end of file
+${BASE_DIRECTORY}/bin/oras push -u AWS -p "${ECR_PASSWORD}" "${IMAGE_REGISTRY}/eks-anywhere-packages-bundles:v1-21-latest" bundle.yaml
+
+# 1.22 Bundle Build
+cd ${BASE_DIRECTORY}/generatebundlefile
+
+${BASE_DIRECTORY}/generatebundlefile/bin/generatebundlefile  \
+    --input ${BASE_DIRECTORY}/generatebundlefile/data/input_122.yaml \
+    --key alias/${KMS_KEY}
+
+cd ${BASE_DIRECTORY}/generatebundlefile/output
+${BASE_DIRECTORY}/bin/oras push -u AWS -p "${ECR_PASSWORD}" "${IMAGE_REGISTRY}/eks-anywhere-packages-bundles:v1-22-${CODEBUILD_BUILD_NUMBER}" bundle.yaml
```

Comment:
We're exposing passwords to the process table here.

Reply:
There are lots of these. I'm making a ticket to deal with this asap after the release. At least these tokens are short-lived and only visible on AWS-controlled machines (and well, the machine of anyone silly enough to clone and run this code).

Reply:
There is an issue about using the ORAS with docker cred helper, but never figured it out.